optaplanner-migration/src/main/resources/META-INF/rewrite/ToLatest9.yml

```diff
@@ -0,0 +1,2777 @@
+type: specs.openrewrite.org/v1beta/recipe
+name: org.optaplanner.migration.ToLatest9
+displayName: Upgrade to the latest OptaPlanner 9
+description: 'Upgrade to the latest OptaPlanner 9 and migrate to Jakarta packages.'
+recipeList:
+  - org.optaplanner.migration.ToLatest
+  - io.quarkus.updates.core.quarkus30.Quarkus3
+  - org.openrewrite.maven.UpgradeDependencyVersion:
+      groupId: org.optaplanner
+      artifactId: optaplanner-bom
+      newVersion: 9.x
```

Comment:
9.x isn't an actual Maven version, is it? How does this work?

Reply:
Right; it takes the latest from the major version 9.

Reply:
Automatically? As in "magic"? Wow, impressive.

Reply:
Yep, the positive outcome of all the artifact resolving Rewrite does under the hood.